dist/common/scripts/scylla_util.py

```diff
@@ -515,10 +510,21 @@ def is_recommended_instance_size(self):
         return False
 
     def is_recommended_instance(self):
-        if self.is_recommended_instance_size() and not self.is_unsupported_instance() and self.is_supported_instance():
+        if self.is_unsupported_instance_class() and self.is_supported_instance_class():
             return True
         return False
 
+    def private_ipv4(self):
+        return self.__instance_metadata("/network/interface/0/ipv4/ipAddress/0/privateIpAddress")
+
+    @staticmethod
+    def check():
```

Comment:
why is a "pass" method needed?

Reply:
It's actually not needed . I found a bug in the code causing some function to use it. removed